cmd/ocm-backplane/cloud/ssm.go

```diff
@@ -126,39 +129,82 @@ func getInstanceID(nodeName string, config *rest.Config) (string, error) {
 }
 
 func startSSMsession(cmd *cobra.Command, argv []string) error {
-	if ssmArgs.node == "" {
-		return fmt.Errorf("--node flag is required")
+	// Check if session-manager-plugin is installed
+	ValidateSessionCmd := ExecCommand("session-manager-plugin", "--version")
+	err := ValidateSessionCmd.Run()
+	if err != nil {
+		return fmt.Errorf("session-manager-plugin is not installed. Please refer AWS doc: https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html")
```

Comment:
small nit: there's a possibility to be other errors. I suggest we can print the error together with the doc. (I didn't test the suggested code, just for an example.)
 ```suggestion
		return fmt.Errorf("failed to validate session-manager-plugin: %w. Please refer AWS doc to make sure session-manager-plugin is properly installed: https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html")
```